commons/c2cgeoportal_commons/models/main.py

```diff
@@ -529,45 +529,176 @@ class LayerV1(Layer):  # Deprecated in v2
+        Enum(
+            'image/jpeg',
+            'image/png',
+            native_enum=False),
+        info={
+            'colanderalchemy': {
+                'title': _('Image type'),
+                'widget': SelectWidget(
+                    values=(
+                        ('', _('- Select -')),
+                        ('image/jpeg', _('image/jpeg')),
+                        ('image/png', _('image/png'))
+                    ))}
+        })  # for WMS
+    style = Column(
+        Unicode,
+        info={
+            'colanderalchemy': {'title': _('Style')}
+        })
+    dimensions = Column(
+        Unicode,
+        info={
+            'colanderalchemy': {'title': _('Dimensions')}
+        })  # for WMTS
+    matrix_set = Column(
+        Unicode,
+        info={
+            'colanderalchemy': {'title': _('Matrix set')}
+        })  # for WMTS
+    wms_url = Column(
+        Unicode,
+        info={
+            'colanderalchemy': {'title': _('Wms url')}
+        })  # for WMTS
+    wms_layers = Column(
+        Unicode,
+        info={
+            'colanderalchemy': {'title': _('Wms layers')}
+        })  # for WMTS
+    query_layers = Column(
+        Unicode,
+        info={
+            'colanderalchemy': {'title': _('Query layers')}
+        })  # for WMTS
+    kml = Column(
+        Unicode,
+        info={
+            'colanderalchemy': {'title': _('Kml')}
+        })  # for kml 3D
+    is_single_tile = Column(
+        Boolean,
+        info={
+            'colanderalchemy': {'title': _('Is single title')}
+        })  # for extenal WMS
+    legend = Column(
+        Boolean,
+        default=True,
+        info={
+            'colanderalchemy': {'title': _('Legend')}
+        })  # on the tree
+    legend_image = Column(
+        Unicode,
+        info={
+            'colanderalchemy': {'title': _('Legend image')}
+        })  # fixed legend image
+    legend_rule = Column(
+        Unicode,
+        info={
+            'colanderalchemy': {'title': _('Legend rule')}
+        })  # on wms legend only one rule
+    is_legend_expanded = Column(
+        Boolean,
+        default=False,
+        info={
+            'colanderalchemy': {'title': _('Is legend expanded')}
+        })
+    min_resolution = Column(
+        Float,
+        info={
+            'colanderalchemy': {'title': _('Min resolution')}
+        })  # for all except internal WMS
+    max_resolution = Column(
+        Float,
+        info={
+            'colanderalchemy': {'title': _('Max resolution')}
+        })  # for all except internal WMS
+    disclaimer = Column(
+        Unicode,
+        info={
+            'colanderalchemy': {'title': _('Disclaimer')}
+        })
     # data attribute field in which application can find a human identifiable name or number
-    identifier_attribute_field = Column(Unicode)
+    identifier_attribute_field = Column(
+        Unicode,
+        info={
+            'colanderalchemy': {'title': _('Identifier')}
```

Comment:
I would prefer "Identifier attribute field" in title. See https://geomapfish-demo.camptocamp.net/1.6/wsgi/admin/LayerV1/109/edit for example.

Reply:
Or "Identifier attribute" or "Identifier field" as attribute and field are redondant.